pkg/config/config.go

```diff
@@ -1,3 +1,3 @@
 package config
 
-var DockerImage = "beryju.org/korb-mover:latest"
+var DockerImage = "korb-mover:latest"
```

Comment:
```suggestion
var DockerImage = "ghcr.io/beryju/korb-mover:latest
```

use GHCR instead as it doesn't have any pull limits

Reply:
Perfect!